NAMESPACE

```diff
@@ -215,6 +211,13 @@ export(tag_rename)
 export(tag_resource)
 export(tag_resource_delete)
 export(tags)
+export(ubuntu_add_swap)
```

Comment:
Can you talk about why the change from debian to ubuntu in these function names? It might make sense, I don't know. If they are changed, we need to make sure when the old function names are used they give a useful message pointing to the new functions. E.g., this guidance https://devguide.ropensci.org/evolution.html#functions-changing-function-names

Reply:
I used a lot of DO products for the thesis, and I even have a DO email since the thesis involved creating a few things such as a CRAN mirror and a dedicated R image. Ubuntu works super well with DO's agent-metrics, a service that integrates well with their cloud dashboard and allows servers monitoring

I'm gonna add dummy functions with deprecation messages, but DO + Ubuntu is a very nice combo

Reply:
Okay, makes sense